dada_tests/interpret/ops/ops_float.dada

```diff
@@ -0,0 +1,79 @@
+async fn main() {
+    if 1.0 + 1.0 == 2.0 {
+        print("1.0 + 1.0 == 2.0").await
+        #! OUTPUT .*
+    }
+
+    if 1.0 - 1.0 == 0.0 {
+        print("1.0 - 1.0 == 0.0").await
+        #! OUTPUT .*
+    }
+
+    if 10.0 * 10.0 == 100.0 {
+        print("10.0 * 10.0 == 100.0").await
+        #! OUTPUT .*
+    }
+
+    if 10.0 / 5.0 == 2.0 {
+        print("10.0 / 5.0 == 2.0").await
+        #! OUTPUT .*
+    }
+
+    # infinity
+    var inf1 = 1.0 / 0.0
+    var inf2 = 2.0 / 0.0
+    if inf1 == inf2 {
+        print("inf1 == inf2").await
+        #! OUTPUT .*
+    }
+
+    # negative infinity
+    var negone = 0.0 - 1.0
+    var negtwo = 0.0 - 2.0
+    var neginf1 = negone / 0.0
+    var neginf2 = negtwo / 0.0
+    if neginf1 == neginf2 {
+        print("neginf1 == neginf2").await
+        #! OUTPUT .*
+    }
+
+    # more negative infinity
+    var negneginf = 0.0 - neginf1
+    if inf1 == negneginf {
+        print("inf1 == negneginf").await
+        #! OUTPUT .*
+    }
+
+    # nan != nan
+    var nan1 = 0.0 / 0.0
+    var nan2 = 0.0 / 0.0
+    if nan1 == nan2 {
```

Comment:
Man, I am so tempted to change this behavior and adopt IEEE total order for floats. =)

Reply:
I'll open an issue.